lib/matplotlib/backends/qt_compat.py

```diff
@@ -180,6 +180,13 @@ def is_pyqt5():
     raise AssertionError("Unexpected QT_API: {}".format(QT_API))
 
 
+# Fixes issues with Big Sur
+# https://bugreports.qt.io/browse/QTBUG-87014, fixed in qt 5.15.2
+if (sys.platform == 'darwin' and
+        LooseVersion(QtCore.qVersion()) < LooseVersion('5.15.2')):
+    os.environ['QT_MAC_WANTS_LAYER'] = '1'
```

Comment:
Can we be a bit more cautious and only set this if it is not already set?

Reply:
I added more conditions.